test/speed-index.js

```diff
@@ -18,6 +18,16 @@ function calculateVisualProgressFromImages(images = [], delay = 1000) {
 	};
 }
 
+test('frame similarity should reflect SSIM', async t => {
+	const data = calculateVisualProgressFromImages([
+		'./assets/frameA.jpg',
+		'./assets/frameC.jpg'
+	]);
+
+	const similarity = speedIndex.calculateFrameSimilarity(data.frames[0], data.frames[1]);
+	t.is(Math.floor(similarity * 100), 78);
```

Comment:
i was able to repro this number on http://darosh.github.io/image-ssim-js/test/browser_test.html with some hacks

the scaling they do on the canvas totally fucks up the numbers. (i end up with 75 instead of 78)
But..  if you breakpoint on `var canvas = document.createElement('canvas');` (inside the img.onload handler) and run this the two times it pauses, you're good: 

```js
width = img.width; height = img.height;
```

screenshot: 
![image](https://cloud.githubusercontent.com/assets/39191/25249831/31dde976-25c8-11e7-83f0-aa400eb2db0e.png)